pkg/cephmgr/osd/scheme.go

```diff
@@ -26,21 +26,29 @@ import (
 )
 
 const (
-	WalPartitionName      = "wal"
-	DatabasePartitionName = "db"
-	BlockPartitionName    = "block"
-	walPartition          = 1
-	databasePartition     = 2
-	blockPartition        = 3
-	UseRemainingSpace     = -1
-	schemeFilename        = "partition-scheme"
-	WalDefaultSizeMB      = 576
-	DBDefaultSizeMB       = 20480
+	Filestore                     = "filestore"
+	Bluestore                     = "bluestore"
+	DefaultStore                  = Filestore
+	WalPartitionName              = "wal"
```

Comment:
Perhaps also group all the partition name as enum? This would make it a bit more organized :)